engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -326,7 +335,7 @@ private void initRenderGraph() {
         renderGraph.addNode(initialPostProcessingNode, "initialPostProcessingNode");
 
         aLabel = "downSampling_gBuffer_to_16x16px_forExposure";
-        DownSamplerForExposureNode exposureDownSamplerTo16pixels = new DownSamplerForExposureNode(context, sceneOpaqueFboConfig, displayResolutionDependentFBOs, FBO_16X16_CONFIG, immutableFBOs, aLabel);
+        DownSamplerForExposureNode exposureDownSamplerTo16pixels = new DownSamplerForExposureNode(context, writeOnlyGBufferConfig, displayResolutionDependentFBOs, FBO_16X16_CONFIG, immutableFBOs, aLabel);
```

Comment:
Okay, I have no clue what we should be doing - I'll have to look at the Nodes again to be certain.
However, as mentioned before, I've only renamed things in this area, and haven't actually changed anything here.